.gitignore

```diff
@@ -1,3 +1,5 @@
 env/
 node_modules/
-TODO.md
\ No newline at end of file
+TODO.md
+
+run.bat
```

Comment:
You added run.bat to .gitignore but included the file anyway. Either is fine, but not both :)